tests/zfs-tests/include/libtest.shlib

```diff
@@ -2059,6 +2059,12 @@ function is_pool_scrub_stopped #pool
 	return $?
 }
 
+function is_pool_scrub_paused #pool
+{
+	check_pool_status "$1" "scan" "scrub paused since "
+	return $?
+}
```

Comment:
You'll want to update the comment above `is_pool_resilvering` to include this additional helper function.  Including changing the 5->6.